superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts

```diff
@@ -137,6 +138,7 @@ export const countries = {
   tanzania,
   thailand,
   timorleste,
+  turkey,
```

Comment:
Do we need to call this Türkiye instead?

Reply:
It has been done.